src/App.tsx

```diff
@@ -6,7 +6,7 @@ import { Library } from 'src/screens/Library'
 import ContextProvider from 'src/state/ContextProvider'
 import ElectronStore from 'electron-store'
 import Sidebar from 'src/components/UI/Sidebar'
-import Login from './screens/Login'
```

Comment:
I'm keeping this here because I might add the old login page as a fallback or in case someone else wants to use it instead of using the webview.